python/tests/tests.py

```diff
@@ -33,7 +33,7 @@ class SparkDLTestCase(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.sc = SparkContext('local[4]', cls.__name__)
+        cls.sc = SparkContext('local[*]', cls.__name__)
```

Comment:
If we do not do that, does it take so long that we get killed? We should make the tests run faster somehow.